tests/cases/conformance/statements/for-inStatements/for-inStatements.ts

```diff
@@ -14,7 +14,7 @@ for (var x in fn()) { }
 for (var x in /[a-z]/) { }
 for (var x in new Date()) { }
 
-var c, d, e;
+var c: any, d: any, e: any;

```

Comment:
The test just needed some values of type `any` but they aren't actually meaningful once we start analyzing control flow.